### PR TITLE
Fix env handling on exec

### PR DESCRIPTION
**Pull request by runcom**
**- What I did**

Mainly, do not append conmon env vars to the container process or conmon's PATH could override container PATH as happened in https://bugzilla.redhat.com/show_bug.cgi?id=1517652

**- How I did it**

vim

**- How to verify it**

check https://bugzilla.redhat.com/show_bug.cgi?id=1517652

**- Description for the changelog**

### CODE REVIEW

File: server/utils.go [hunk 1]
```diff
@@ -210,3 +212,44 @@ func validateLabels(labels map[string]string) error {
...
+				continue
+			}
+			envs = append(envs, item.GetKey()+"="+item.GetValue())
+		}
+		if imageConfig != nil {
+			for _, imageEnv := range imageConfig.Config.Env {
+				var found bool
+				parts := strings.SplitN(imageEnv, "=", 2)
+				if len(parts) != 2 {
+					continue
```
Comment (rhatdan) [inline server/utils.go]: Are any of these errors?  Is this expected?

Comment (runcom) [reply server/utils.go]: nope, just ignore like docker does

**Pull request merged**